sdks/techbooks-node/src/core.ts

```diff
@@ -776,7 +776,7 @@ declare const Deno: any;
 declare const EdgeRuntime: any;
 type Arch = 'x32' | 'x64' | 'arm' | 'arm64' | `other:${string}` | 'unknown';
 type PlatformName =
-  | 'MacOS'
+  | 'macOS'
```

Comment:
This is an API break. I'm not really expecting it to be accepted, but it seems worth noting that it's misspelled especially given that `iOS` is spelled correctly.